net/dawn/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# This software is licensed under the Public Domain.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dawn
+PKG_VERSION:=2020-03-23
```

Comment:
Replace with PKG_SOURCE_DATE

Reply:
Hmmm, I did the same in libuwifi? I have to change it their too? Or will that break everything?

Reply:
No. Just cosmetic.